test_abnormality_detection.py

```diff
@@ -42,9 +47,13 @@ def parse_args() -> argparse.Namespace:
     image = Image.open(args.image_path).convert('RGB')
     # load allowed_regions map
     map_ = Image.open(args.allowed_regions).convert('L')
+    map_rgb = Image.open(args.allowed_regions)
     allowed_regions = np.asarray(map_)
```

Comment:
It is better to only open the image once, Also, consider change the name from `map_rgb` to `allowed_regions_img` (Remember to Update the references for `map_rgb` to `allowed_regions_img` later in the code as well)
```suggestion
    allowed_regions_img= Image.open(args.allowed_regions)
    allowed_regions = np.asarray(allowed_regions_img.convert('L'))
```